WordPress/Classes/ViewRelated/System/Notices/NoticeStyle.swift

```diff
@@ -53,6 +53,18 @@ public struct QuickStartNoticeStyle: NoticeStyle {
     public let layoutMargins = UIEdgeInsets(top: 13.0, left: 16.0, bottom: 13.0, right: 16.0)
 
     public let isDismissable = false
+
+    static func titleFont() -> UIFont {
+        let font = WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .semibold)
+        if #available(iOS 11.0, *) {
+            // setting adjustsFontForContentSizeCategory=true is not working with fonts with custom fontWeight
+            // so we are using UIFontMetrics to get it work
+            let metrics = UIFontMetrics(forTextStyle: .subheadline)
+            return metrics.scaledFont(for: font)
```

Comment:
It seems that we are scaling an already scaled font.
`font` has the proper pointSize for the "current" dynamic size. Then we ask the system to scale it again.
This strategy works well when the font given to the `UIFontMetrics` has (always) the proper size for the **default** dynamic size configuration (15 in the case of `subheadline`)

Reply:
It seems like we are making the same mistake here:

https://github.com/wordpress-mobile/WordPress-iOS-Shared/blob/ccc3244543248c29243e682e49e8f75e11378190/WordPressShared/Core/Views/WPStyleGuide%2BDynamicType.swift#L76

😱

Reply:
I am removing this altogether, returning new font instances seems to solve our problems